server/model/display-type.js

```diff
@@ -0,0 +1,8 @@
+// @flow
+export type DisplayType = 
+  | "lead"
+  | "regular";
```

Comment:
Yeah, I think makes sense. Although, weight is for the images in articles right? and DisplayType refers to the Promos in Transporters. Does it make sense to combine them?